proposal/package-importer.d.ts.md

```diff
@@ -216,17 +216,27 @@ using symlinks if this behavior is desired.
 
 [ECMAScript modules]: https://nodejs.org/api/esm.html#no-node_path
 
+The [Node resolution algorithm] requires a `parentURL`, used for determining
+where in the file system to start searching for a module. When compiling a
+string, for instance `compileString('@use "pkg:bootstrap";')`, we don't know
+where to start looking for the Bootstrap module. We considered
+`require.main.filename` and the current working directory, but found that
+neither would allow for all use cases. We decided to allow users to specify an
+entry point path, defaulting to `require.main.filename`.
+
 ## Types
 
 ```ts
 import {FileImporter, Importer} from '../spec/js-api/importer';
 ```
 
-### `nodePackageImporter`
+### `NodePackageImporter`
 
 ```ts
-export declare const nodePackageImporter: unique symbol;
-export type NodePackageImporter = typeof nodePackageImporter;
+export interface NodePackageImporter {
+  pkgImporter: 'node';
+  entryPointPath?: string;
+}
```

Comment:
I think we should make this a proper class, so that users call it as:

```js
sass.compileString(stylesheet, {
  importers: [new NodePackageImporter(entrypoint)]
});
```

Reply:
Updated in https://github.com/sass/sass/pull/3748/commits/3afb76cc2bf5df211040bfb51323f4e5a0fd074e